modules/server/distribution/pom.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@kaviththiranga Better to include License header

Reply:
Fixed